layouts/404.html

```diff
@@ -1,7 +1,7 @@
 {{ define "main" }}
   {{ $content :=  "The requested page was not found. <a href='/' class='has-text-weight-bold'>Go home?</a>" | safeHTML }}
-  {{ $dict := dict "class" "is-danger" "title" "Not Found" "subtitle" "Error 404" "content" $content }}
-  {{ partial "component/message-block.html" $dict }}
+  {{ $dict := dict "color" "215 0 0" "title" "Not Found" "subtitle" "Error 404" "content" $content }}
```

Comment:
Old versions of Hugo didn't allow multiline dictionaries. Go ahead break this one up to make it readable:

```
{{ $dict := dict 
  "color" "215 0 0" 
  "title" "Not Found"
  "subtitle" "Error 404" 
  "content" $content 
}}
```

Reply:
Also the other ones like this.